docs/developers/node-tutorial.md

```diff
@@ -4,49 +4,106 @@
 In this tutorial, we will cover how to use the Celestia Node API to submit and
 retrieve messages from the Data Availability Layer by their namespace ID.
 
+This tutorial was assumes you are working in a Linux environment.
+
 > To view a video tutorial for setting up a Celestia Light Node, click [here](./light-node-video.md)
 
 ## Hardware Requirements
 
-You can find hardware requirements [here](../nodes/light-node.md#hardware-requirements).
+The following minimum hardware requirements are recommended for running a light node:
+
+- Memory: 2 GB RAM
+- CPU: Single Core
+- Disk: 5 GB SSD Storage
+- Bandwidth: 56 Kbps for Download/56 Kbps for Upload
 
 ## Setting Up Dependencies
 
-You can follow the tutorial for setting up the dependencies [here](./environment.md).
+First, make sure to update and upgrade the OS:
```

Comment:
This means they're still building on Ubuntu btw

Reply:
Should I comment this somewhere? 

We mention both YUM and APT package managers, should we elaborate?